rebalance.py

```diff
@@ -3,12 +3,11 @@
 r.gSystem.Load('libRooFit')
 import numpy as np
 
-
 @dataclass(frozen=True)
 class Jet():
     pt: float
-    eta: float
     phi: float
+    eta: float
```

Comment:
let's please keep the ordering of 'pt, eta, phi'. The solution to the problem you found earlier is not to change the constructor, but to just use it with explicit keyword arguments to preclude any chance of mismatches:
```python
myjet = Jet(pt=10., eta=2.1, phi=0.3)
```